cmd/chain-maind/app/app.go

```diff
@@ -272,6 +272,11 @@ func newApp(logger log.Logger, db dbm.DB, traceStore io.Writer, appOpts serverty
 	}
 
 	snapshotDir := filepath.Join(cast.ToString(appOpts.Get(flags.FlagHome)), "data", "snapshots")
+	err = os.MkdirAll(snapshotDir, os.ModePerm)
```

Comment:
not created by SDK?

Reply:
for the cosmos sdk in chain-main v2, it is required for rocksdb